csrc/scheduler/matmul.cpp

```diff
@@ -453,9 +453,10 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
       "scheduleMatmul supports fusion with single mma op in definition, got ",
       mma_ops.size());
 
-  TensorView* a = roles_map.at(MatmulRole::MMA_INPUT_A);
-  TensorView* b = roles_map.at(MatmulRole::MMA_INPUT_B);
-  TensorView* c = roles_map.at(MatmulRole::MMA_OUTPUT);
+  // Core roles: there can be only one... TV with assigned core role
+  TensorView* a = roles_map.at(MatmulRole::INPUT_A).front();
+  TensorView* b = roles_map.at(MatmulRole::INPUT_B).front();
+  TensorView* c = roles_map.at(MatmulRole::OUTPUT_D).front();
```

Comment:
This looks strange `c = ....(D)`. I understand that if we rename to `d = ...(D)`, this will be a huge refactor. So I am fine with this inconsistency for now.

Reply:
You are right, I though about updating the variable names and documentation to fix the inconsistency, but as you pointed out, it would blur what is the key change for this feature.

I plan to have a follow up PR where I will ask @liqiangxl to review, as it will impact his changes related to epilogue being stored in shared memory before pushing data to global tensor.